.github/workflows/sims.yml

```diff
@@ -3,7 +3,7 @@ name: Sims
 # This workflow will run on all Pull Requests, if a .go, .mod or .sum file have been changed
 on:
   schedule:
-    - cron: "* */2 * * *"
```

Comment:
this one should run every couple of hours so we know things fail as things are bing merged